WEB_first project/ASP.NET CORE Project#1/Models/Account.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ASP.NET_CORE_Project_1.Models
+{
+    [Table(nameof(Account))]
+    public class Account
+    {
+        [Key]
+        public int Id { get; set; }
+        public string? UserId { get; set; }
```

Comment:
If you want to join User table on this field, you should mark it as foreign key and set the type for it as Guid